controllers/storagecluster/reconcile.go

```diff
@@ -144,6 +146,19 @@ func (r *StorageClusterReconciler) Reconcile(ctx context.Context, request reconc
 	defer func() { r.Log = prevLogger }()
 	r.Log = r.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 
+	oc := &apiv1.OperatorCondition{}
+	err := r.Client.Get(ctx, types.NamespacedName{Name: "ocs-operator.v4.9.0", Namespace: request.Namespace}, oc)
+	condition, err := NewUpgradeable(r.Client)
+
+	fmt.Println(oc.Spec)
```

Comment:
These seem like a lot of  individual  Println calls. I think we can skip some of them

Reply:
yes, I ll remove them. These were just added by me for debugging.